README.md

```diff
@@ -0,0 +1,59 @@
+### OMERO api Gradle plugin
+
+The _omero-api-plugin_ is a gradle plugin that provides users and projects the ability to generate/compile the files
+required to use _omero-blitz_
+
+From a high level, blitz-plugin consists of the following tasks/stages:
```

Comment:
Is this now about omero-blitz-plugin?

Reply:
I think they are reversed. I'll push a commit for review.